chapters/03_queueing_model/01_discrete_event_simulation/main.tex

```diff
@@ -0,0 +1,281 @@
+\subsection{Discrete Event Simulation}
+
+Discrete event simulation (DES) is currently one of the most popular modelling
+techniques.
+It is a method for modelling the behaviour of real-world systems in which the
+system is made up of discrete events, each of which has a certain
+duration~\cite{DESstewart}.
+It can be used to understand complex situations in order to make predictions
+and thus provide improvements~\cite{VinceGeraintBook}.
+There three main approaches to building DES models are the activity scanning
+approach, the event scheduling approach and the process interaction
+approach~\cite{DESapproaches}.
+Under the scope of this study only the event scheduling approach is considered.
+This section describes the discrete event simulation (DES) model used to
+represent the queueing network of Section~\ref{sec:queueing-section}.
+
+In order to use DES on the queueing network described in
+Figure~\ref{fig:diagram_of_queueing_system} some slight modifications are
+required.
+The current queueing network is a two-node queueing system that accepts two
+types of individuals, where type 1 individuals arrive at waiting zone 1 and
+type 2 individuals arrive at waiting zone 2.
+The modification that is required revolves around the mechanisms of waiting
+zone 2.
+Waiting zone 2 is defined as a non-service node where there is only a queueing
+space for individuals to wait there until they are allowed to waiting zone 1.
+From an implementation perspective there is an equivalent system that can be
+used where instead of a node with no service and queueing capacity \(M\), there
+are \(M\) servers each serving with a service rate of \(0\) and no queueing
+capacity.
+
+\input{chapters/03_queueing_model/img/equivalent_queueing_model_diagram.tex}
+
```

Comment:
Make this a figure.

In the caption include a sentence along the lines of:

```
Note that there is no waiting zone at the first service node.
```

Reply:
This is already a figure. The current caption is:

```latex
An equivalent model to the one described in
Figure~\ref{fig:diagram_of_queueing_system}. The difference between the two
diagrams is the formulation of waiting zone 2. The original diagram uses
a node with no servers and a queueing capacity of \(M\) while this one uses
\(M\) servers with no queueing capacity.
```